### PR TITLE
AT-40 Local token storage

### PR DESCRIPTION
**Pull request by berabulut**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines. For the guidelines check the README.
- [ ] Tests for the changes have been added (for bug fixes / features). As we discussed this is optional but encouraged.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR lets us store our encrypted access token in browser. We are using AES algorithm to encrypt it.

* **What is the current behavior?** (You can also link to an open issue here)
Currently we are hard coding our access token to make api calls to spotify.

* **What is the new behavior (if this is a feature change)?**
We don't need to hard code our access token. We can safely store our token in browser and access it when we need it.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:
I recommend merging this after "36-styledNavBar" and "37-Iframe". Because this includes other PRs work too.  And I will delete what is conflicting with other PRs. I am keeping them for now to let you test.

### CODE REVIEW

**Comment by csjoblinksreddit** [comment]
Can you create a ticket for this and then put it in the title as well.

File: frontend/src/components/home.js [hunk 1]
```diff
@@ -2,40 +2,64 @@ import React, { Component } from 'react';
...
+          else {
+            localStorage.removeItem('key')
+            localStorage.removeItem('access_token')
+            localStorage.removeItem('refresh_token')
+          }  
+        }
+      }
     }
-
+    /*TODO:
```
Comment (csjoblinksreddit) [inline frontend/src/components/home.js]: Does this comment have to be here anymore?

File: frontend/src/components/home.js [hunk 2]
```diff
@@ -2,40 +2,64 @@ import React, { Component } from 'react';
...
-        .then((response) => {
-          console.log(response.item)
-          if(response){
-            this.setState({
-              nowPlaying: {
-                name: response.item.name,
-                albumArt: response.item.album.images[0].url
-              }
-            });
+    const params = this.getHashParams();
```
Comment (csjoblinksreddit) [inline frontend/src/components/home.js:18]: This is a lot of stuff. What do you think about moving all of the key code into a method of its own and then calling that method inside of componentdidmount?

**Review by csjoblinksreddit — CHANGES_REQUESTED** [review]
I see that you made some changes after this review. Is this pr ready for a review?

**Review by csjoblinksreddit — COMMENTED** [review]
Overall great job! Just want to address some issues.

File: frontend/src/components/home.js (hunk 2 above)
Comment (berabulut) [reply frontend/src/components/home.js:18]: I've tried to slice it as much as i can.

**Review by csjoblinksreddit — APPROVED** [review]

**Pull request merged**